src/validation.cpp

```diff
@@ -3040,7 +3047,7 @@ bool ContextualCheckBlock(const CBlock& block, CValidationState& state, const Co
     //   {0xaa, 0x21, 0xa9, 0xed}, and the following 32 bytes are SHA256^2(witness root, witness nonce). In case there are
     //   multiple, the last one is used.
     bool fHaveWitness = false;
-    if (VersionBitsState(pindexPrev, consensusParams, Consensus::DEPLOYMENT_SEGWIT, versionbitscache) == THRESHOLD_ACTIVE) {
+    if (VersionBitsState(pindexPrev, consensusParams, Consensus::DEPLOYMENT_SEGWIT2X, versionbitscache) == THRESHOLD_ACTIVE) {
```

Comment:
This would partition the p2p network due to the [unexpected-witness DOS ban](https://github.com/btc1/bitcoin/blob/687b3cd2fecaf38bb38a103b05c55805df85755b/src/validation.cpp#L3069-L3076) .